scripts/server.py

```diff
@@ -550,66 +567,544 @@ def handle_client(conn, addr):
+                                f"BirdWeather Submission Error:: Detection POST - Connection Timeout! Max retries of {bw_request_max_retries} reached - {conn_timeout_exec}",
+                                flush=True)
+                    except requests.exceptions.ConnectionError as conn_err_exec:
+                        if submission_retries < bw_request_max_retries:
+                            can_retry = True
+                            submission_retries += 1
+                            # Calculate the backoff time before making another request
+                            request_backoff_time = bw_request_backoff_factor * (2 ** (submission_retries - 1))
+
+                            print(
+                                f"BirdWeather Submission Error:: Soundscape POST - Connection Error! Retrying after {request_backoff_time}s, Retry Attempt:{submission_retries} - {conn_err_exec}",
+                                flush=True)
+                        else:
+                            print(
+                                f"BirdWeather Submission Error:: Soundscape POST - Connection Error! Not Retrying - {conn_err_exec}",
+                                flush=True)
+                    except requests.exceptions.RequestException as request_exc:
+                        # Check if the status code is one that we can try on
+                        req_status_code = request_exc.response.status_code
+                        if req_status_code in bw_request_status_retry_for:
+                            # Check if we're still table to retry
+                            if submission_retries < bw_request_max_retries:
+                                can_retry = True
+                                submission_retries += 1
+                                request_backoff_time = bw_request_backoff_factor * (2 ** (submission_retries - 1))
+
+                                if debug_birdweather_submissions:
+                                    print(
+                                        f"BirdWeather Submission Error:: Detection POST - HTTP Request Exception for {soundscape_url}, Retrying after {request_backoff_time}s, Retry Attempt:{submission_retries} - {request_exc.response.reason}",
+                                        flush=True)
+                                else:
+                                    print(
+                                        f"BirdWeather Submission Error:: Detection POST - HTTP Request Exception for {soundscape_url}, Retrying after {request_backoff_time}s, Retry Attempt:{submission_retries}",
+                                        flush=True)
+                        else:
+                            print(
+                                f"BirdWeather Submission Error:: Detection POST - HTTP Request Exception for {soundscape_url}, Not retrying - Retries Attempted:{submission_retries} - {request_exc}",
+                                flush=True)
+                    except (
```

Comment:
related to the comment above. If you agree that we don't need to decode the response json, then we don't need to treat this here.